test/CMakeLists.txt

```diff
@@ -23,7 +18,7 @@ target_include_directories(unit_tests PRIVATE
 target_link_directories(unit_tests PRIVATE ${ORT_LIB_DIR})
 target_link_libraries(unit_tests PRIVATE
   onnxruntime-genai-static
-  GTest::gtest_main
```

Comment:
note: there's already a `main` in `test/main.cpp` so we don't need one from gtest